src/style.css

```diff
@@ -0,0 +1,152 @@
+
+body {
+  margin-top: 0;
+  margin-bottom: 0;
+  font-weight: 300;
+  padding-top: 20px ;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir.ttc);
+}
+
+@font-face {
+  font-family: Avenir-Book;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir-Heavy";
```

Comment:
Don't use different names for the same font. It should be `font-family: "Avenir"` but with rule: `font-weight: 900`